ui/raidboss/data/06-ew/raid/p7s.ts

```diff
@@ -25,6 +46,90 @@ const triggerSet: TriggerSet<Data> = {
         text: 'Split Tankbusters',
       },
     },
+    {
+      id: 'P7S Spark of Life',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7839', source: 'Agdistis', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: 'Raidwide + Bleed',
```

Comment:
```suggestion
        text: 'aoe + bleed',
```

Mostly just to match other examples of aoe+ bleed elsewhere.  I know people say "raidwide", but cactbot says "aoe" for all of these, and it's better to be consistent.